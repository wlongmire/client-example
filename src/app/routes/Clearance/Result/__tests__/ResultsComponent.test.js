/* eslint-env node, jest */

import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import { Result } from '../index'

jest.mock('./../../../../actions/userActions', () => 'testActions')
jest.mock('./../../../../actions/submissionActions', () => 'testActions')

process.env.SUPPRESS_NO_CONFIG_WARNING = true
// console.log('SUPPRESS_NO_CONFIG_WARNING: ' + config.util.getEnv('SUPPRESS_NO_CONFIG_WARNING'));


const initialState = {
  result: {},
  input: {
    insuredAddress: {
      primaryInsuredAddress: 'Test 999',
      primaryInsuredState: 'New York',
      primaryInsuredCity: 'Test 999',
      primaryInsuredZipcode: 'Test 999',
    },
    primaryInsuredName: 'Test 999',
    projectAddress: {
      projectAddress: 'Test 999',
      projectCity: 'Test 999',
      projectState: 'California',
      projectZipcode: 'Test 999'
    }
  },
  submission: {
    type: 'oi'
  },
  user: {
    broker: 'test-broker-id'
  }

}

// Snapshot for Home Header Component
describe('>>> Clearance Result Component--- Snapshot', () => {
  it('+++capturing Snapshot with', () => {
    const renderedValue = shallow(
      <Result
        input={initialState.input}
        result={initialState.result}
        submission={initialState.submission}
        handleSubmit={jest.fn()}
        user={initialState.user}
      />)
    expect(toJson(renderedValue)).toMatchSnapshot()
  })
})
