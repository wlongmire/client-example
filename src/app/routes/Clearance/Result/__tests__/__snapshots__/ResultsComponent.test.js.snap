// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`>>> Clearance Result Component--- Snapshot +++capturing Snapshot with 1`] = `
<form>
  <h3>
    This Submission Has Passed Clearance!
  </h3>
  <div>
    <h4>
      You are the first to submit this insured for review. Now we can enter additional pricing information.
    </h4>
    <div
      className="matchContainer"
    >
      <div
        className="match"
      >
        <h4>
          First Named Insured:
        </h4>
        <h5>
          Test 999
        </h5>
        <h4>
          Project Address:
        </h4>
        <h5>
          Test 999
          <br />
          Test 999
           
          California
           
          Test 999
        </h5>
      </div>
    </div>
  </div>
  <ButtonGroup
    block={false}
    bsClass="btn-group"
    justified={false}
    vertical={false}
  >
    <Button
      active={false}
      block={false}
      bsClass="btn"
      bsStyle="default"
      className="btn clearanceButton"
      disabled={false}
      onClick={[Function]}
    >
      Fill out Remaining Information
    </Button>
    <LinkContainer
      action="push"
      onlyActiveOnIndex={false}
      to="/productChoice"
    >
      <Button
        active={false}
        block={false}
        bsClass="btn"
        bsStyle="default"
        className="btn secondary"
        disabled={false}
      >
         Return to Product Selection
      </Button>
    </LinkContainer>
  </ButtonGroup>
</form>
`;
